pkg/network/cache/multi.go

```diff
@@ -324,9 +324,14 @@ func (x *multiClient) Close() error {
 func (x *multiClient) RawForAddress(addr network.Address) *rawclient.Client {
 	x.mtx.Lock()
 
-	c := x.createForAddress(addr).Raw()
+	strAddr := addr.String()
+
+	c, cached := x.clients[strAddr]
+	if !cached {
```

Comment:
It is sensible to place this logic inside the `createForAddress`

Reply:
We can do small refactor there. Check https://github.com/nspcc-dev/neofs-node/pull/694/commits/ab78fa1830b935cc877e152b13ef0d6310abb360

I would like to keep `createForAddress` only for creation of new clients. To get or create we can use `x.client(addr)`.